README.md

```diff
@@ -1,25 +1,108 @@
 # Quarkus Backstage
 
-[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.backstage/quarkus-backstage?logo=apache-maven&style=flat-square)](https://search.maven.org/artifact/io.quarkiverse.backstage/quarkus-backstage)
+[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.backstage/quarkus-backstage?logo=apache-maven&style=flat-square)](https://central.sonatype.com/artifact/io.quarkiverse.backstage/quarkus-backstage-parent)
 
-## Welcome to Quarkiverse!
+Generate Backstage Catalog Information as part of the Quarkus build or the Quarkus CLI.
 
-Congratulations and thank you for creating a new Quarkus extension project in Quarkiverse!
+## Features
 
-Feel free to replace this content with the proper description of your new project and necessary instructions how to use and contribute to it.
+- Generate the catalog-info.yaml for the Quarkus application
+- Command Line interface to install / uninstall and Component to Backsatge
+- Orchestrate Quarkus extensions:
+  - kubernetes
+  - helm
+  - argocd
 
-You can find the basic info, Quarkiverse policies and conventions in [the Quarkiverse wiki](https://github.com/quarkiverse/quarkiverse/wiki).
+## Requirements
 
-In case you are creating a Quarkus extension project for the first time, please follow [Building My First Extension](https://quarkus.io/guides/building-my-first-extension) guide.
+### For Catalog Info Generation
+To generate the catalog-info.yaml nothing special is required.
```

Comment:
This sentence is irrelevant if nothing is needed

Reply:
I think that it is needed, cause things are required for other features of the extension (e.g. tokens etc). It should be clear that this is not the case when generating `catalog-info.yaml`